exense-commons-parent/exense-basic-commons/src/main/java/ch/exense/commons/io/FileWatchService.java

```diff
@@ -60,18 +62,21 @@ public void run() {
 
 
 			try {
+				List<Entry<File, Subscription>> subscriptionsList;
 				synchronized (subscriptions) {
```

Comment:
@cl-exense If we do it this way I assume we could remove the synhronized block and use a ConcurrentHashMap instead